docs/designers-developers/developers/tutorials/javascript/plugins-background.md

```diff
@@ -0,0 +1,17 @@
+
+## Plugins Background
+
+The primary means of extending WordPress remains the plugin, see [Plugin Basics](https://developer.wordpress.org/plugins/the-basics/) for more. The quick start is to create a new directory in `wp-content/plugins/` to hold your plugin code, for our example you can call it `myguten-plugin`.
+
+Inside of this new directory, create a file called `myguten-plugin.php` which will be the server-side code the runs when your plugin is active. For now you can place the following in that file:
+
+```php
+<?php
+/*
+Plugin Name: My Guten Plugin
+*/
+```
```

Comment:
Should this comment use [DocBlock](https://en.wikipedia.org/wiki/PHPDoc#Components_of_PHPDoc) instead? Like

```
/**
 * Plugin Name: My Guten Plugin
 */

```
My eyes seem to be more used to this kind of comments when it comes to WordPress plugins, but I'm unsure if that's a recommendation written anywhere.

Reply:
Either works, but the standard is `/*` for the header: https://developer.wordpress.org/plugins/the-basics/header-requirements/

Reply:
I'll leave as standard, thanks @TimothyBJacobs